src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+.smallForm,
+
+.form {
+  position: relative;
+}
+
+.form {
+  margin: 20px 0;
+}
```

Comment:
Follow this example, please 
<img width="1665" alt="image" src="https://user-images.githubusercontent.com/99403198/222885789-420c0eba-a9ed-45f9-81de-9bc77c7b413d.png">